lib/content_type.mli

```diff
@@ -82,6 +82,9 @@ module Parameters : sig
   type t = value Map.t
   (** Type of parameters. *)
 
+  val of_list : (key * value) list -> t
+  (** Make {!t} from an association list. *)
+
   val key : string -> key option
   (** [key v] makes a new key (according to RFC 2045). *)
```

Comment:
under what circumstances does `key` return a `None` here? worth clarifying in docstring

Reply:
When it does not respect RFC 2045, we allow some characters as a key but not all.

Reply:
might be better to make this a `Rresult` error so that the reason for the error isnt lost